openwisp_network_topology/tests/test_selenium.py

```diff
@@ -76,6 +76,12 @@ def setUp(self):
             organization=org,
         )
 
+    def tearDown(self):
+        # Workaround to clear the web driver console logs
+        # after the invocation of each test method
+        # https://stackoverflow.com/questions/44991009/clear-chrome-browser-logs-in-selenium-python
```

Comment:
This is one of those cases in which we can move this link to the commit message since I don't see a lot of benefit in leaving it in the code.